### ISSUE TITLE
install-ableC-bundle fails

### ISSUE THREAD

**Issue by ericvanwyk** [melt-umn/ableC-vm-artifact#5]
This script uses the master branch and, now, fails.  Is this not tested in Jenkins?  It is supposed to be the "easy" way to try out ableC.

I can run this by changing the branch-name variable to be "develop" and then is seems to work.  It fails on the cilk stuff since I've not installed those libraries where I'm testing this.

Any reason to not change this to just use "develop"?  

On one hand, I would like to see us using the workflow where we merge develop into master periodically.  But changing to "develop" is an easy way to get this working...

**Comment by krame505** [comment]
Yeah, the way this was originally intended is that the bundle is something stable-ish, corresponding to the master branch.  The short-term fix is to just do a new release of everything.  Longer-term, maybe we could consider having Jenkins set up to do weekly releases, automatically merging and tagging everything in master?  @tedinski any thoughts?

**Comment by tedinski** [comment]
No thoughts really. Why does master fail?

I thought the idea was this was something stable. We shouldn't be able to break it without committing something to master. So how did it get broken? We shouldn't have committed anything to master...

**Comment by tedinski** [comment]
Oh, at a guess: we released a new version of silver, but not ablec/etc?

Probably it should be changed to checking out specific tagged releases known to work together.

**Comment by krame505** [comment]
It looks like we are using the latest jars to compile an older version of silver-ableC, which fails because we are trying to compile against a newer version of the reflection library.  Not really sure what the alternative is, but presumably we should be fetching some older set of jars corresponding to the Silver release?

**Comment by ericvanwyk** [comment]
The first error was that in ``master`` on ableC the ``edu.umn.cs.melt.ableC`` grammar is *not* in a ``grammars`` directory.  But on ``develop`` it is.  The ``build.sh`` scripts in some of the directories in the ``ableC_sample_projects`` repos assumes that it is in a ``grammars`` directory and appends ``grammars`` to a path.

If we can limit the jars to ones for ``master`` and ``develop`` that would be OK, but I'd hate to see a proliferation of jars for different branches.  Maybe also jars for the ``silver-ableC`` mix would OK...

**Comment by ericvanwyk** [comment]
Things in develop do occasionally break, so we do need to be merging into master more frequently.   The install script clones Silver from master, but the Silver install web page doesn't specify the branch in the git clone command, thus getting it from develop.  

1. I will change the install script to use develop since that has a much better chance of working than master does right now.
2. Yes, let's do a merge into master of things so that we can change this back soon.
3. We should be giving proper version numbers to things.  I don't think we are doing that now.  This is mostly for archival reasons.  I would like to be able to say precisely in papers what versions of things were used so that people could, in theory, replicate what we've claimed to have done.

What version numbers are we on now for Silver, Copper, ableC?
